### PR TITLE
chore(ui-react-storage): Add e2e tests for Storage Manager

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Description of changes

This change adds integration tests for Storage Manager component.

#### Description of how you validated changes

`yarn e2e dev`

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by reesscot — DISMISSED** [review]
LGTM

File: packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-multipart.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+Feature: Upload a file to S3 with public access level settings
```
Comment (zchenwei) [inline packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-multipart.feature:2]: Should we make the name distinct from the default one?

Comment (ioanabrooks) [inline packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-multipart.feature:2]:
```suggestion
Feature: Storage Manager with default files
```

Comment (ioanabrooks) [reply packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-multipart.feature]: Good catch!

File: examples/next/pages/ui/components/storage/storage-manager/private-access-level/index.page.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  useAuthenticator,
+  withAuthenticator,
+} from '@aws-amplify/ui-react';
+import { StorageManager } from '@aws-amplify/ui-react-storage';
+import '@aws-amplify/ui-react/styles.css';
+import awsExports from './aws-exports';
+Amplify.configure(awsExports);
+
+export function StorageManagerExample() {
+  const { signOut } = useAuthenticator();
```
Comment (wlee221) [inline examples/next/pages/ui/components/storage/storage-manager/private-access-level/index.page.tsx:13]:
```suggestion
  const { signOut } = useAuthenticator({ signOut } => [signOut]);
```
for optimizations

File: packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-default-files.feature [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (wlee221) [inline packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-default-files.feature:1]:
```suggestion
```

**Review by zchenwei — APPROVED** [review]

File: packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-multipart.feature [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+Feature: Storage Manager with default files
```
Comment (wlee221) [inline packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-multipart.feature:2]:
```suggestion
Feature: Storage Manager with default files
```

**Review by wlee221 — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]

**Pull request merged**